### PR TITLE
add Pop OS Logo

### PR DESCRIPTION
**Pull request by matteomiceli**
simpleicons

**Issue:** #3578
**Alexa rank:** 67,180
  

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description

I chose the colour most prominent in logo, the hex value came from webpage stylesheet https://pop.system76.com/.

### CODE REVIEW

**Comment by runxel** [comment]
Hi @matteomiceli!
Thanks for your PR 💯

As you can see the build is currently failing – but not because there is a problem with your icon, instead it seems the `!` makes troubles with the filename.
By the way: You can use spaces in the brandname! I would rather expect it to be `"title": "Pop! OS"`.
As stated in the [CONTRIBUTING.md](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#svg-filename-convention) file you can always check which is the correct file name for a given string (brand name). In this case it should be `popos.svg`.

**Comment by runxel** [comment]
Scratch the last command. They _really_ want to have the underscore 🤦‍♂️
Still: remove the exclamation mark, please!

**Comment by matteomiceli** [comment]
> Scratch the last command. They _really_ want to have the underscore 🤦‍♂️
> Still: remove the exclamation mark, please!

Thanks for the response. Should I remove the '!' in the .svg title data as well as the object in the simple-icons.json file?

**Comment by runxel** [comment]
No, you can leave it in the `.json`. There it should actually be.
And the underscore stays in the filename, then the build will work.
So in the json: `"title": "Pop!_OS"`
filename: `pop_os.svg`

Sorry for the confusion!

File: icons/pop_os.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...021831-2.004931-2.064954c0-1.300944.886707-2.105165 2.032713-2.105165 1.328903.01182 1.977149 1.019467 1.965572 2.093338zm-1.965572-1.438135c-.858925 0-1.340479.610263-1.345109 1.442865 0 .723799.395892 1.402656 1.328903 1.402656.93301 0 1.289545-.735626 1.289545-1.409751.01157-.700145-.351905-1.419213-1.273339-1.43577zm2.539733 2.982713c0 .153748.0046-.730894 0-.865719h.629724l.04399.541664c.20605.120634.525543.210516.787156.210516.486184 0 .993204-.125362.993204-.610261-.01621-.439957-.581105-.529834-1.037192-.577145-.77558-.08042-1.400674-.274383-1.400674-1.163754 0-.792395.877447-1.187412 1.530323-1.187412.525542-.01182.898283.07333 1.37289.461246 0 .319324-.0046.508548-.0046.827873h-.608886l-.03936-.525109c-.245407-.12063-.49776-.153745-.692233-.149016-.419045.0047-.865871.205789-.865871.565324 0 .352433.289395.449416.803361.525106C23.192008 17.91895 24 18.112908 24 19.025935c-.0046.99818-.921435 1.260728-1.70859 1.260728-.599627.0023-.974684-.111165-1.449292-.534563zM15.42 19.943693c0 .182132-.14354.328786-.321807.328786h-3.720465c-.178268 0-.321808-.146654-.321808-.328786v-.02838c0-.182127.14354-.328785.321808-.328785h3.720465c.178267 0 .321807.146658.321807.328785z"/></svg>
```
Comment (adamrusted) [inline icons/pop_os.svg:1]: This looks great! Is there any chance you could run this SVG through [SVGOMG](https://jakearchibald.github.io/svgomg/) or run `npm run svgo -- icons/pop_os.svg` on this, just to try and reduce the 'precision' (number of decimals) down to 3?
For example, here's my resulting path running it through SVGOMG (bear in mind I've not tested this!).
```svg
M21.446 13.104a.718.718 0 01-.301.456c-.255.163-.565.116-.785-.083-.206-.184-.276-.496-.21-.766.066-.281.28-.53.552-.608.61-.172.862.504.744 1zm-.373-1.92c-.378-.155-.248-2.177.097-4.632.097-.686.264-1.01.452-1.176.268-.234.757-.243 1.088-.194.38.057.743.237 1.018.509.243.241.271.487.188.806-.137.509-.37.996-.595 1.472a51.626 51.626 0 01-.581 1.128c-1.123 2.02-1.348 2.216-1.667 2.086zm-7.617-.912c.222.97.199 2.035.01 2.801-.179.724-.663 1.372-1.432 1.59-.673.191-1.458.052-1.94-.422-.791-.775-1.255-2.597-1.048-3.919.166-1.062.727-2.422 1.956-2.481.079-.005.157-.003.236.005 1.253.115 1.935 1.194 2.218 2.426zm-2.264 2.207c.07.012.139.012.208-.01.234-.068.368-.316.43-.557a1.968 1.968 0 00-.041-1.121 1.2 1.2 0 00-.296-.492c-.498-.466-.996.018-1.082.57-.067.43.03.93.295 1.277.122.16.298.3.486.333zm4.975-.151c-.002 1.79.174 3.16-.597 3.323-.625.13-.605-.43-.653-.9-.185-1.825-.354-4.855-.426-6.688.05-.787.567-1.27 1.104-1.388.829-.182 1.713-.13 2.447.333 1.452.918 1.568 2.602.688 3.976-.482.75-1.938 1.294-2.563 1.247zm.185-4.014c-.09.014-.238.057-.28.147-.1.21-.093 1.547.081 2.107.028.092.09.18.18.213.33.123.626-.289.76-.537.188-.345.248-.752.13-1.13-.111-.353-.456-.864-.87-.8zm-8.487.89c-.544 1.175-1.447 2.11-2.697 2.57.358.958.724 1.905 1.08 2.86.2.516.39 1.081.216 1.59-.18.52-.884.664-1.396.11C4.084 15.267.758 8.492.512 7.984.303 7.545-.01 7.077 0 6.57c.016-.762 1.17-1.533 1.723-1.928.773-.551 1.667-.927 2.623-.93.745 0 1.382.208 1.947.577a3.991 3.991 0 011.69 2.498c.194.901.153 1.824-.118 2.415M5.33 7.917a3.75 3.75 0 00-1.188-1.853 1.286 1.286 0 00-.4-.248c-1.038-.348-.589 1.424-.42 1.871.234.627.589 1.42 1.063 1.892a.755.755 0 00.368.225c.216.045.415-.109.523-.288.09-.15.125-.327.146-.5.04-.366.005-.74-.092-1.1zm14.938 10.29c-.005 1.039-.625 2.077-1.993 2.077s-2.005-1.022-2.005-2.065c0-1.3.886-2.105 2.032-2.105 1.33.012 1.978 1.02 1.966 2.094zm-1.966-1.438c-.859 0-1.34.61-1.345 1.443 0 .724.396 1.403 1.33 1.403.932 0 1.289-.736 1.289-1.41.011-.7-.352-1.42-1.274-1.436zm2.54 2.983c0 .154.005-.73 0-.866h.63l.044.542c.206.12.525.21.787.21.486 0 .993-.125.993-.61-.016-.44-.58-.53-1.037-.577-.776-.08-1.4-.274-1.4-1.164 0-.792.877-1.187 1.53-1.187.525-.012.898.073 1.373.461 0 .32-.005.509-.005.828h-.609l-.04-.525a1.506 1.506 0 00-.692-.149c-.419.005-.865.206-.865.565 0 .353.289.45.803.525.838.114 1.646.308 1.646 1.22-.005 1-.921 1.262-1.709 1.262-.6.002-.974-.112-1.449-.535zm-5.422.192a.325.325 0 01-.322.328h-3.72a.325.325 0 01-.322-.328v-.029c0-.182.143-.328.322-.328h3.72c.178 0 .322.146.322.328z
```

**Comment by adamrusted** [comment]
Changes have been made by @matteomiceli - could another @simple-icons/maintainers look at this being as I created the path?

**Review by fbernhart — CHANGES_REQUESTED** [review]
I don't think that we can merge this pull request the way it is right now.

The used [SVG from Wikimedia](https://commons.wikimedia.org/wiki/File:Pop_OS-Logo-nobg.svg) seems to be a bit outdated. While the "_os" part is under the word "Pop!" on Wikimedia, on [their website](https://pop.system76.com/) they use the two words side by side.

The "not stacked" version can as well be seen [here](https://64.media.tumblr.com/77cb82385581cb2144b05e7c001e4e66/ec1935ac7bf54dd4-81/s500x750/8a0fce62c90b805044c7e513d9701d9bd46ba311.png) and [here](https://www.facebook.com/system76/photos/a.202298187728/10160777492812729/?type=3&theater), while I can't find any official logo where the two words are stacked.

So I would suggest updating the `source` attribute to https://pop.system76.com and rebuilding the logo from the SVG embedded in their website, which is this one:

```
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1499 549"><path d="M926.2 397.2c-1.9 7.9-6 14.9-13 19.3-11 6.9-24.4 4.9-33.9-3.5-8.9-7.8-11.9-21-9.1-32.4 2.9-11.9 12.1-22.4 23.9-25.7 26.3-7.3 37.2 21.3 32.1 42.3zM910.1 316c-16.3-6.5-10.7-92 4.2-195.8 4.2-29 11.4-42.7 19.5-49.7 11.6-9.9 32.7-10.3 47-8.2 16.4 2.4 32.1 10 44 21.5 10.5 10.2 11.7 20.6 8.1 34.1-5.9 21.5-16 42.1-25.7 62.2-2.3 4.9-22.4 43-25.1 47.7-48.5 85.4-58.2 93.7-72 88.2z" fill="#6CC7D2"></path><path d="M581.1 277.5c9.6 41 8.6 86 .4 118.4-7.7 30.6-28.6 58-61.8 67.2-29.1 8.1-63 2.2-83.8-17.8-34.2-32.8-54.2-109.8-45.3-165.7 7.2-44.9 31.4-102.4 84.5-104.9 3.4-.2 6.8-.1 10.2.2 54.1 4.9 83.6 50.5 95.8 102.6zm-97.8 93.3c3 .5 6 .5 9-.4 10.1-2.9 15.9-13.4 18.6-23.6 4.1-15.5 3.5-32.2-1.8-47.4-2.7-7.7-6.7-15.2-12.8-20.8-21.5-19.7-43 .8-46.7 24.1-2.9 18.2 1.3 39.3 12.7 54 5.3 6.8 12.9 12.7 21 14.1z" fill="#BBB5B3"></path><path d="M698.2 364.4c-.1 75.7 7.5 133.6-25.8 140.5-27 5.5-26.1-18.2-28.2-38.1-8-77.1-15.3-205.2-18.4-282.7 2.2-33.3 24.5-53.7 47.7-58.7 35.8-7.7 74-5.5 105.7 14.1 62.7 38.8 67.7 110 29.7 168.1-20.8 31.7-83.7 54.7-110.7 52.7v4.1zm8-169.7c-3.9.6-10.3 2.4-12.1 6.2-4.3 8.9-4 65.4 3.5 89.1 1.2 3.9 3.9 7.6 7.8 9 14.2 5.2 27-12.2 32.8-22.7 8.1-14.6 10.7-31.8 5.6-47.8-4.8-14.9-19.7-36.5-37.6-33.8z" fill="#99918D"></path><path d="M339.6 232.3C316.1 282 277.1 321.5 223.1 341c15.5 40.5 31.3 80.5 46.7 120.9 8.6 21.8 16.8 45.7 9.3 67.2-7.8 22-38.2 28.1-60.3 4.7C176.3 488.6 32.6 202.2 22 180.7 13 162.2-.5 142.4-.1 121c.7-32.2 50.5-64.8 74.4-81.5C107.7 16.2 146.3.3 187.6.2c32.2 0 59.7 8.8 84.1 24.4 37.4 24 63.4 62.3 73 105.6 8.4 38.1 6.6 77.1-5.1 102.1m-109.5-54.4c-8.4-30.5-27.4-58.6-51.3-78.3-5.2-4.3-10.8-8.3-17.3-10.5-44.8-14.7-25.4 60.2-18.1 79.1 10.1 26.5 25.4 60 45.9 80 4.5 4.4 9.8 8.3 15.9 9.5 9.3 1.9 17.9-4.6 22.6-12.2 3.9-6.3 5.4-13.8 6.3-21.1 1.7-15.5.2-31.3-4-46.5z" fill="#48B9C7"></path><path d="M1337.8 460.8c-.2 43.9-27 87.8-86.1 87.8-59.1 0-86.6-43.2-86.6-87.3 0-55 38.3-89 87.8-89 57.4.5 85.4 43.1 84.9 88.5zm-84.9-60.8c-37.1 0-57.9 25.8-58.1 61 0 30.6 17.1 59.3 57.4 59.3s55.7-31.1 55.7-59.6c.5-29.6-15.2-60-55-60.7zm109.7 126.1c0 6.5.2-30.9 0-36.6h27.2l1.9 22.9c8.9 5.1 22.7 8.9 34 8.9 21 0 42.9-5.3 42.9-25.8-.7-18.6-25.1-22.4-44.8-24.4-33.5-3.4-60.5-11.6-60.5-49.2 0-33.5 37.9-50.2 66.1-50.2 22.7-.5 38.8 3.1 59.3 19.5 0 13.5-.2 21.5-.2 35h-26.3l-1.7-22.2c-10.6-5.1-21.5-6.5-29.9-6.3-18.1.2-37.4 8.7-37.4 23.9 0 14.9 12.5 19 34.7 22.2 36.2 4.8 71.1 13 71.1 51.6-.2 42.2-39.8 53.3-73.8 53.3-25.9.1-42.1-4.7-62.6-22.6z" fill="currentcolor"></path><path d="M1128.4 534.2c0 7.7-6.2 13.9-13.9 13.9H953.8c-7.7 0-13.9-6.2-13.9-13.9V533c0-7.7 6.2-13.9 13.9-13.9h160.7c7.7 0 13.9 6.2 13.9 13.9v1.2z" fill="#FAA41A"></path></svg>
```

**Comment by adamrusted** [comment]
@fbernhart done!

**Comment by fbernhart** [comment]
@adamrusted LGTM! 👍🏼  And thanks to @matteomiceli for the initial work on this icon. 🥇

**Pull request merged**